boot/zephyr/CMakeLists.txt

```diff
@@ -54,6 +54,8 @@ macro(app_set_runner_args)
   endif()
 endmacro()
 
+include($ENV{ZEPHYR_BASE}/../nrf/cmake/boilerplate.cmake NO_POLICY_SCOPE)
```

Comment:
Can we get a comment about why this line is needed? I'm guessing it's because it's an out of tree board definition?